app/views/directives/_copy-to-clipboard.html

```diff
@@ -1,11 +1,17 @@
-<div class="input-group copy-to-clipboard">
-  <input id="{{id}}" type="text" class="form-control" value="{{clipboardText}}" ng-disabled="isDisabled" ng-readonly="!isDisabled" select-on-focus>
+<div class="input-group copy-to-clipboard" style="{{displayWide ? 'max-width:100%;' : ''}}">
+  <input id="{{id}}" type="text" class="form-control" value="{{inputText ? inputText : clipboardText}}" ng-disabled="isDisabled" ng-readonly="!isDisabled" select-on-focus>
   <span class="input-group-btn" ng-hide="hidden">
-    <a data-clipboard-target="#{{id}}"
+    <a ng-show="!inputText" data-clipboard-target="#{{id}}"
```

Comment:
If you use `ng-if` here and line 10, does it let you remove the `get(0)` / `get(1)` special case in the controller?